public/v2/apps/phpldapadmin.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/osixia/docker-openldap",
+    "description": "OpenLDAP with administration interface",
+    "displayName": "OpenLDAP + phpLDAPadmin",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "ports": ["389:389", "636:636"],
+                "environment": {
+                    "LDAP_ORGANISATION": "$$cap_openldap_ldap-organisation",
+                    "LDAP_DOMAIN": "$$cap_openldap_ldap-domain",
+                    "LDAP_ADMIN_PASSWORD": "$$cap_openldap_ldap-admin-password"
+                }
+            },
+            "php$$cap_appnameadmin": {
+                "image": "osixia/phpldapadmin:$$cap_phpldapadmin_version",
+                "containerHttpPort": "443",
+                "restart": "always",
+                "depends_on": ["$$cap_appname"],
+                "environment": {
+                    "PHPLDAPADMIN_TRUST_PROXY_SSL": true,
+                    "PHPLDAPADMIN_LDAP_HOSTS": "$$cap_appname.$$cap_root_domain",
```

Comment:
Wouldn't `srv-captain--$$cap_appname` work too? If it works it'll be a better alternative as it uses an internal connection.

Reply:
It works so I change :)